x-pack/plugins/viz_editor/common/query_types.ts

```diff
@@ -63,6 +86,8 @@ export interface TermsOperation extends FieldOperation {
   operator: 'terms';
   argument: Field & {
     size: number;
+    orderBy?: number;
```

Comment:
Why is `orderBy` a number? If it's referencing the ID of another operation, it should be `orderById`

Reply:
Yeah, this is a bit dirty - it references to a position in the esaggs query. But since we have an id now on each operation in our select array, it easy to map that correctly. Changed it to reference to the select operation instead of the esaggs agg